### ISSUE TITLE
[Tokyo Mirage Sessions] 60 FPS mod causes infinite loading when navigating from "Artists" to the main menu, after clicking into the characters cards. Characters also do not load properly within.

### ISSUE THREAD

**Issue by BlueGrassBlade** [ActualMandM/cemu_graphic_packs#506]
When applying the 60 FPS mod for the game, going into the "Artists" tab from the in-game menu and clicking into the characters causes the game to not load them. Just a menu with a space that's obviously meant for the characters and their "mirages". Exiting from this menu and trying to go back to the main in-game menu causes infinite loading. This issue only happens when 60 FPS mod is enabled across all Vsync options including the new experimental one. However, at the original 30 FPS, this does not happen and works as intended.

Steps to reproduce the behaviour:
1. Boot game up and get past title screen (loading a save game).
2. Go to in-game menu by pressing Y button (or whatever key that you configured).
3. Select "Artists".
4. Click on any character cards.
5. Issue: characters are not loaded with their mirages.
5. Try to go back to the main in-game menu (prior to the "Artists" tab).
6. Issue: infinite loading.

Expected behaviour
1. Characters and their mirages should be properly loaded upon selecting the character cards.
2. Going back to the main in-game menu should not have infinite loading and display the menu accordingly.

Screenshots
1. 60 FPS character card loading issue
![image](https://user-images.githubusercontent.com/79953477/109744836-edbdf380-7c0d-11eb-8c08-ad5aa0d026d4.png)

2. Correct behaviour at 30 FPS
![image](https://user-images.githubusercontent.com/79953477/109745328-a8e68c80-7c0e-11eb-83ed-3a404b42a24f.png)

Desktop (please complete the following information):
 - OS: Windows 10 Pro
 - GPU: NVIDIA RTX 2060
 - Renderer: Vulkan
 - Version: 1.22.7

Video showing issues
https://user-images.githubusercontent.com/79953477/109746190-f6172e00-7c0f-11eb-875a-9b46a6b1ace1.mp4

**Comment by BlueGrassBlade** [comment]
Happens in Cemu version 1.22.8 as well.

**Comment by ActualMandM** [comment]
Does this issue still happen with the latest graphic pack versions?

**Comment by edgar1016** [comment]
> Does this issue still happen with the latest graphic pack versions?

Yes it does I'm running CEMU 1.26.2f and the latest graphics pack.

This also happens in the first 2 minutes of the game when you talk to Tsubasa right at the start she her sprite won't load and the game locks up sound it still running though. Turning off 60FPS fixes all the issues.
